README.md

```diff
@@ -1,3 +1,32 @@
 # Child
-
 [![Child](https://circleci.com/gh/Ethylix/child.svg?style=svg)](https://circleci.com/gh/Ethylix/child)
+
+Child is an IRC Service for [UnrealIRCD](https://www.unrealircd.com).
+
+# Dependencies
+- `libmysqlclient` (or `libmariadbclient-dev-compat`)
+- `libssl`
+- `libc`
+- `cmake`
+- `check`
+- mysql 5.7
+- unrealircd 4.2
+
+# Build
+```
+apt install libmariadb-dev-compat libmariadbclient-dev libc-dev libssl-dev cmake check
+mkdir build
+cd build
+cmake ..
+make
+```
+
+For development, you can use docker-compose to set an environment up: [doc/docker.md](doc/docker.md).
+
+# Usage
+```
+./child
+```
+
+# Configuration
+Child expects a configuration file in the same directory as the binary: `child.conf`. You can find an example in [doc/example.conf](doc/example.conf).
```

Comment:
The configuration file is expected to be in the current working directory, not necessarily the binary's directory.